pkg/maps/ctmap/entry.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2016-2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ctmap
+
+import (
+	"fmt"
+	"unsafe"
+
+	"github.com/cilium/cilium/pkg/byteorder"
+)
+
+// CtEntry represents an entry in the connection tracking table.
+type CtEntry struct {
```

Comment:
This patch backport would be much simpler if the `Slave` field is just added to the existing CtEntry structure instead. Then the backport commit would also match line-for-line the commit being backported.